addons/sys_core/initSettings.sqf

```diff
@@ -27,3 +27,13 @@
     false,
     {["disableUnmuteClients", _this] call FUNC(setPluginSetting)}
 ] call CBA_Settings_fnc_init;
+
+[
+    QGVAR(disableGearDesync),
```

Comment:
I'd keep that as `QGVAR(disableDesyncHint)`, don't want people thinking it could in any way disable actual desync.